o/opentelemetry-ruby/opentelemetry-ruby_ubi_8.5.sh

```diff
@@ -19,9 +19,9 @@
 # ----------------------------------------------------------------------------
 
 PACKAGE_NAME="opentelemetry-ruby"
-PACKAGE_VERSION=${1:-"opentelemetry-sdk-experimental/v0.1.0"}
```

Comment:
@vinodk99 so this build script will not work for v0.1.0 now?. We need to add incremental support for new version, keeping support for old version also. So that through Currency we can run build for earlier supported version also.

Reply:
@nikhil-kalbande , it will work but it requires latest ruby version so i have updated the build-script with latest version